eclipse/ide-target-platform/category.xml

```diff
@@ -25,7 +25,6 @@
    <bundle id="org.eclipse.jetty.servlet" version="0.0.0"/>
    <bundle id="org.eclipse.jetty.server" version="0.0.0"/>
    <bundle id="org.eclipse.jetty.util" version="0.0.0"/>
-   <bundle id="org.hamcrest" version="1.1.0.v20090501071000"/>
```

Comment:
@chanseokoh here's the problem: I removed the `org.hamcrest` 1.1 bundle from `category.xml` but failed to add the 1.3 bundles.